Master andromeda

MissKira opened the pull request:
:changelog:
- add: Добавлены Прототипы и РСИ
- remove: Убраны лишние части прототипов
- tweak: Изменены убранные части прототипов
- fix: Исправлена некоторая часть брони у вещей

MissKira committed:
Add files via upload

Добавила свою папочку для РСИ, набор комбинезонов СССП, 1 вагон и вагонетку РСИ для медипенов.

MissKira committed:
Очень-Очень большое обновление. (СССП + СПБсТ)

Добавила огромное количество прототипов для медипенов, добавила startinggear с евой и просто, множество прототипов(заполненных) вещей. ну и совсем капельку прототипов для СССП

MissKira committed:
Просто иконка СПБсТ

Да.

MissKira committed:
Ребаланс Медипенов Спецназа

Небольшой ребаланс сделанный на основе тестов Никфореста.

MyFriendHowAreYou commented on Resources/Textures/Interface/Misc/job_icons.rsi/meta.json:
```diff
@@ -1,155 +1,158 @@
+            "name": "Zookeeper"
+        }
+    ]
+}
```
Кир, удали этот весь кринж нахуй и просто добавь новый meta.json в папку andromeda, ну или сделай пометочку хавку через # или // о том что это твоё что бы он понял из за чего серв крашица

MissKira committed:
Delete Resources/Prototypes/Andromeda/Kira's Prototypes directory

MissKira committed:
Большое удаление и большое добавление.

Удалила старую папку с прототипами, загрузила по новой, но уже рассортированную и с множеством новых добавок, в целом произошел некоторый ребаланс.

MissKira committed:
Add files via upload

MissKira committed:
Небольшая добавка 

Добавила обычные перчатки спецназа, не предназначенные для драк.

MissKira committed:
Удаление проблемного yml,

Для последующего ребаланса и изменения.

MissKira committed:
Удаление проблемного рси.

Для последующей переработки.

MissKira committed:
Небольшое добавление

Вернула бронежилет, только пофикшенный

MissKira committed:
Небольшое добавление

Фикс текстурки для Бронежилета спецназа, и новое расположение его соответственно

MissKira committed:
Ребаланс и Добавление

Ребаланс защиты у многих вещей, она где-то снижена где-то повышена соответствуя логике покрытия и так далее.
В основном именно уменьшена.
Добавлено 2 новых типа бронежилета для спецназа, и переработано собственно в плане защиты.

MissKira committed:
RSI для бронежилетов и новых перчаток

Помимо бронежилетов были добавлены дополнительно не боевые перчатки спецназа, они не обладают никакой защитой кроме как от электричества.

MissKira committed:
RSI Fix

MissKira committed:
RSI Fix

MissKira committed:
RSI Fix

MissKira committed:
Добавила РПС медика

MissKira committed:
Добавила РПС медика

MissKira committed:
Delete SwatWedding.yml

MissKira committed:
Поменяла StartingGear для Медика

Ну и собственно заполнила РПСку вещами.
